### PR TITLE
update holochain script

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	make HC_REV=$(HC_REV) update-hc-sha
+	git checkout -b update-hc-$(HC_REV)
+	git add nixpkgs.nix
+	git commit -m hc-rev:$(HC_REV)
+	git push origin HEAD
+
+update-hc-sha:
+	@if [ $(HC_REV) ]; then\
+		echo "⚙️  Updating holo-envoy using holochain rev: $(HC_REV)";\
+		echo "✔  Updating holo-nixpkgs rev in nixpkgs.nix...";\
```
Comment (Connoropolous) [inline Makefile]: these seem out of place

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		echo "✔  Replacing rev...";\
+		sed -i '3s/.*/REV=$(HC_REV)/' ./install-holochain.sh;\
+	else \
+		echo "No holo-nixpkgs rev provided"; \
```
Comment (Connoropolous) [inline Makefile]: This also seems like a misleading comment

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	git commit -m hc-rev:$(HC_REV)
+	git push origin HEAD
+
+update-hc-sha:
+	@if [ $(HC_REV) ]; then\
+		echo "⚙️  Updating conductor-api using holochain rev: $(HC_REV)";\
+		echo "✔  Replacing rev...";\
+		sed -i '3s/.*/REV=$(HC_REV)/' ./install-holochain.sh;\
+	else \
+		echo "No holo-nixpkgs rev provided"; \
```
Comment (zippy) [inline Makefile]:
```suggestion
		echo "No holochain rev provided"; \
```

**Review by zippy — APPROVED** [review]

**Review by zippy — CHANGES_REQUESTED** [review]
the script also needs to update the holochain rev values in the README in default.nix, and in test/e2e/fixutres/zomes/foo/Cargo.toml

**Review by zippy — APPROVED** [review]

**Pull request merged**